tests/Aspire.Hosting.RabbitMQ.Tests/RabbitMQFunctionalTests.cs

```diff
@@ -141,11 +147,13 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
                 {
                     var hb = Host.CreateApplicationBuilder();
                     hb.Configuration[$"ConnectionStrings:{rabbitMQ2.Resource.Name}"] = await rabbitMQ2.Resource.ConnectionStringExpression.GetValueAsync(default);
+                    hb.Services.AddXunitLogging(testOutputHelper);
                     hb.AddRabbitMQClient(rabbitMQ2.Resource.Name);
 
                     using (var host = hb.Build())
                     {
                         await host.StartAsync();
+                        await app.WaitForTextAsync($"Time to start RabbitMQ:", resourceName: "rabbitMQ").WaitAsync(TimeSpan.FromMinutes(1));
```

Comment:
`$"Time to start RabbitMQ:"` should be a constant since you are using it in multiple places